php/pantheon/checks/cron.php

```diff
@@ -78,6 +78,11 @@ public function message(Messenger $messenger) {
         // @TODO move this logic to the run() function or checkCron() function
 				if ($this->cron) {
         	foreach ($this->cron as $timestamp => $crons) { 
+            // Truncate the crons list so we're not creating unreasonably large reports
+            // @TODO: A lot of cron jobs may be a sign of trouble and should probably be a warning.
+            if (count($crons) < 100) {
```

Comment:
Would it be appropriate to a) let the user know that we are truncating the list or b) that they more than a healthy amount?  If there is no good use-case / it's actually bad practice, we could just not send back crons and send back a 'whoaa there, 10k crons?  really?' message.

cc @joshkoenig